src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -177,12 +177,18 @@ internal static bool DoSslHandshake(SafeSslHandle context, byte[] recvBuf, int r
         {
             sendBuf = null;
             sendCount = 0;
+            int retVal;
```

Comment:
nit: you don't need to move where retVal is declared... you could just make the below code be:
```C#
if (BioWrite(context.InputBio, recvBuf, recvOffset, recvCount) <= 0)
{
    ...
}
```